src/main/java/bungae/thunder/cakey/common/error/exception/ErrorCode.java

```diff
@@ -0,0 +1,40 @@
+package bungae.thunder.cakey.common.error.exception;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
+public enum ErrorCode {
+
+    // Common
+    INVALID_INPUT_VALUE(400, "C001", "Invalid Input Value"),
+    METHOD_NOT_ALLOWED(405, "C002", "Invalid Input Value"),
+    ENTITY_NOT_FOUND(400, "C003", "Entity Not Found"),
+    INTERNAL_SERVER_ERROR(500, "C004", "Server Error"),
+    INVALID_TYPE_VALUE(400, "C005", "Invalid Type Value"),
```

Comment:
`INVALID_INPUT_TYPE`이 더 좋지 않을까?
```suggestion
    INVALID_INPUT_TYPE(400, "C005", "Invalid Input Type"),
```

Reply:
좋은 생각